webpack/assets/javascripts/react_app/components/SearchBar/index.js

```diff
@@ -1,25 +1,9 @@
 import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
-import * as actions from '../AutoComplete/AutoCompleteActions';
-import reducer from '../AutoComplete/AutoCompleteReducer';
 import SearchBar from './SearchBar';
+import { selectAutocompleteSearchQuery } from '../AutoComplete/AutoCompleteSelectors';
 
-const mapStateToProps = ({
-  autocomplete: {
-    error, results, searchQuery, status,
-  },
-}) => ({
-  error,
-  results,
-  searchQuery,
-  status,
+const mapStateToProps = ({ autocomplete }, { autocomplete: { id } }) => ({
+  searchQuery: selectAutocompleteSearchQuery(autocomplete, id),
```

Comment:
I afraid it is difficult to understand that the first `autocomplete` is from the `state` and the second `autocomplete` is from the props.

Do you mind changing to:
```js
const mapStateToProps = (state, props) => ({
  searchQuery: selectAutocompleteSearchQuery(state, props.autocomplete.id),
});
```

I think we should use it as a convention.